semantic_conventions/trace/messaging.yaml

```diff
@@ -180,3 +180,44 @@ groups:
         required:
           conditional: 'If missing, it is assumed to be false.'
         brief: 'A boolean that is true if the message is a tombstone.'
+  
+  - id: cloudevents
+    prefix: cloudevents
+    brief: > 
+      Attributes for CloudEvents
+    attributes:
+      - id: id
+        type: string
+        brief: >
+          The [unique identifier](https://github.com/cloudevents/spec/blob/master/spec.md#id) of the event within the scope of the producer.
+        examples: ['123e4567-e89b-12d3-a456-426614174000', 'producer-1']
+      - id: source
+        type: string
+        brief: >
+          The [source](https://github.com/cloudevents/spec/blob/master/spec.md#source-1) that identifies where the event happened.
+        examples: ['com.github.pull_request.opened', 'com.example.object.deleted.v2']
+      - id: specversion
+        type: string
+        brief: >
+          The [version of the CloudEvents specification](https://github.com/cloudevents/spec/blob/master/spec.md#specversion) which the event uses.
+        examples: '1.0'
+      - id: type
+        type: string
+        brief: >
+          The [type](https://github.com/cloudevents/spec/blob/master/spec.md#type) of event.
+        examples: ['com.github.pull_request.opened', 'com.example.object.deleted.v2']
+      - id: datacontenttype
```

Comment:
The [attribute naming conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md) says that multi-words should be separated with a `_`. But, in the ones defined in cloudevents it's all together. Not sure what approach to take, change it to follow the OTel name convention or keep as is? 🤔